test/service.spec.ts

```diff
@@ -190,6 +191,14 @@ describe('DebuggerService', () => {
           }
         };
       });
+      // create the list of source breakpoints expected from a restore
+      sourceBreakpoints = breakpoints.map(breakpoint => {
+        return {
+          line: breakpoint.line,
+          verified: breakpoint.verified,
+          source: breakpoint.source
```

Comment:
Ah, looks like this change is needed in this code base, because of this: https://github.com/jupyterlab/debugger/pull/556

And the fact they have diverged quite a bit because of the strict null checks in core but not here...